202009.0/acd288bc-2a1a-429d-b0fb-8a73fa6b6a6f.md

```diff
@@ -20,12 +20,14 @@ YOUR ADVANTAGES:
 
 To install Inxmail run the command in the console:
 ```php
- composer require spryker-eco/inxmail:1.0.0
+ composer require spryker-eco/inxmail:1.1.0
  ```
 
 ## Configuration
 
-To set up the Inxmail initial configuration, use the credentials you received from your Inxmail server. Space id, key id and secret you can get from Settings → API keys panel on Inxmail server:
```

Comment:
from the https://help.inxmail-commerce.com/inxmail-commerce-api.html:

`https://{spaceid}.api.inxmail-commerce.com/api-service/`

> The space ID is specified as first part of the domain name.